dev-libs/aws-c-common/aws-c-common-0.5.2.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2020 Gentoo Authors
```

Comment:
How did you manage to push a 2020-copyrighted file with repoman??

Reply:
Strange... I always do a "repoman full" beforehand, no idea how that slipped...